controller/src/beerocks/master/db/db.cpp

```diff
@@ -3599,3 +3599,16 @@ void db::clear_vap_list()
 }
 
 const std::shared_ptr<db::vaps_list_t> db::get_vap_list() { return m_vap_list; }
+
+bool db::is_prplmesh(const std::string &mac)
+{
+    auto node = get_node(mac);
+    return node ? node.get()->is_prplmesh : false;
```

Comment:
Maybe I should have added a debug output here to signal that the node is not in the DB yet and we read `is_prplmesh` too early.

Reply:
Fixed.